### PR TITLE
Populate new fields from `outport.AlteredAccount` structure

### PR DESCRIPTION
**Pull request by miiu96**
## Reasoning behind the pull request
- Populate the news field from the `outport.AlteredAccount` structure
- Bug-fix case when some logs are missing in case of a built-in function call with smart contract call 
  
## Proposed changes
-  Upgraded the version of `elastic-indexer-go` module: https://github.com/ElrondNetwork/elastic-indexer-go/pull/194
- Upgraded the version of `elrond-go-core` module: https://github.com/ElrondNetwork/elrond-go-core/pull/142

## Testing procedure
- 

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/ElrondNetwork/elrond-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -4,7 +4,7 @@ go 1.17
 
 require (
 	github.com/ElrondNetwork/elastic-indexer-go v1.3.5
-	github.com/ElrondNetwork/elrond-go-core v1.1.28
+	github.com/ElrondNetwork/elrond-go-core v1.1.29-0.20221221101642-8a1c79726c3e
```
Comment (iulianpascalau) [inline go.mod:7]: let's not forget about proper release

File: outport/process/alteredaccounts/alteredAccountsProvider.go [hunk 2]
```diff
@@ -140,6 +137,25 @@ func (aap *alteredAccountsProvider) processMarkedAccountData(
...
 
+func (aap *alteredAccountsProvider) getAlteredAccountFromUserAccounts(userEncodedAddress string, userAccount state.UserAccountHandler) *outportcore.AlteredAccount {
+	alteredAccount := &outportcore.AlteredAccount{
+		Address:  userEncodedAddress,
+		Balance:  userAccount.GetBalance().String(),
+		Nonce:    userAccount.GetNonce(),
+		UserName: string(userAccount.GetUserName()),
+	}
+
+	if core.IsSmartContractAddress(userAccount.AddressBytes()) {
```
Comment (iulianpascalau) [inline outport/process/alteredaccounts/alteredAccountsProvider.go:148]: does this works also on a metachain SC?

File: outport/process/alteredaccounts/alteredAccountsProvider_test.go [hunk 3]
```diff
@@ -1202,3 +1227,19 @@ func getMockArgs() ArgsAlteredAccountsProvider {
 		EsdtDataStorageHandler: &testscommon.EsdtStorageHandlerStub{},
 	}
 }
+
+type myStruct struct {
+	message string
+}
+
+func (m myStruct) String() string {
+	return "belesti-magarul"
```
Comment (iulianpascalau) [inline outport/process/alteredaccounts/alteredAccountsProvider_test.go:1236]: what is this?

File: process/smartContract/process.go [hunk 4]
```diff
@@ -1399,6 +1401,10 @@ func (sc *scProcessor) processIfErrorWithAddedLogs(acntSnd state.UserAccountHand
 	}
 
 	processIfErrorLogs := make([]*vmcommon.LogEntry, 0)
+	if prevVmOutput != nil && len(prevVmOutput.Logs) > 0 {
+		processIfErrorLogs = append(processIfErrorLogs, prevVmOutput.Logs...)
```
Comment (iulianpascalau) [inline process/smartContract/process.go:1405]: so this will essentially merge the logs

File: go.mod (hunk 1 above)
Comment (miiu96) [reply go.mod:7]: removed

Comment (miiu96) [reply go.mod:7]: will add.

File: process/smartContract/process.go (hunk 4 above)
Comment (miiu96) [reply process/smartContract/process.go:1405]: exactly

File: outport/process/alteredaccounts/alteredAccountsProvider.go (hunk 2 above)
Comment (miiu96) [reply outport/process/alteredaccounts/alteredAccountsProvider.go:148]: yes, is working also from the metachain smart contracts.

**Review by iulianpascalau — COMMENTED** [review]
:+1:

**Review by sasurobert — DISMISSED** [review]

**Review by iulianpascalau — COMMENTED** [review]
:+1:

**Review by sasurobert — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by popenta — APPROVED** [review]
System test passed.

**Pull request merged**